swan-lake/development-tutorials/observability/overview-of-ballerina-observability.md

```diff
@@ -0,0 +1,204 @@
+
+        if !products.hasKey(orderRequest.productId.toString()) {
+            log:printError("Product not found with product ID: " + orderRequest.productId.toString());
+            http:NotFound errorResponse = {
+                body:  "Product not found with product ID: " + orderRequest.productId.toString()
+            };
+            return errorResponse;
+        }
+        Product product = products.get(orderRequest.productId.toString());
+        Order newOrder = {orderId: orderCount, productId: orderRequest.productId, quantity: orderRequest.quantity, totalPrice: product.price * orderRequest.quantity};
+        orders[orderCount.toString()] = newOrder;
+        orderCount += 1;
+
+        log:printInfo("Order placed successfully. " + newOrder.toString());
+        http:Accepted response = {
+            body:  newOrder.toJson()
+        };
+        return response;
+    }
+
+    // Get order details by ID.
+    resource function get 'order/[int orderId]() returns http:Ok|http:NotFound|error? {
+        log:printInfo("Fetching order details");
+
+        if (!orders.hasKey(orderId.toString())) {
+            log:printError("Order not found with order ID: " + orderId.toString());
+            http:NotFound errorResponse = {
+                body:  "Order not found with order ID: " + orderId.toString()
+            };
+            return errorResponse;
+        }
+
+        Order 'order =  <Order> orders[orderId.toString()];
+        log:printInfo("Order details fetched successfully. " + 'order.toString());
+        http:Ok response = {
+            body:  'order.toJson()
+        };
+        return response;
+    }
+}
+```
+
+### Enable observability for a Ballerina project
+Observability can be enabled in a Ballerina project by adding the following section to the `Ballerina.toml` file. 
+```toml
+[build-options]
+observabilityIncluded=true
+```
+
+>**Note:** The above configuration is included by default in the `Ballerina.toml` file generated when initiating a new 
+package using the `bal new` command.
+
+Alternatively, we can pass the `--observability-included` flag with the `bal run` command to start a Ballerina program with observability enabled.
+
+### Setting up Ballerina runtime configurations
+To enable observability (both metrics and tracing) in the Ballerina runtime, use the following configurations:
+```toml
+[ballerina.observe]
+enabled = true
+provider = <PROVIDER>
+```
+
+Metrics and tracing can be enabled separately as well by using the following configurations. Add additional configurations specific to the tool or platform you are using.
+```toml
+[ballerina.observe]
+metricsEnabled=true
+metricsReporter=<METRICS_REPORTER>
+tracingEnabled=true
+tracingProvider=<TRACING_PROVIDER>
+```
+
+Configuration key | Description | Default value | Possible values 
+--- | --- | --- | --- 
+ballerina.observe. metricsEnabled | Whether metrics monitoring is enabled (true) or disabled (false) | false | `true` or `false`
```

Comment:
```suggestion
`ballerina.observe.metricsEnabled` | Whether metrics monitoring is enabled (true) or disabled (false) | false | `true` or `false`
```

Reply:
Check all the other occurrences and add quotes.

Reply:
Addressed review suggestions